cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/BasicMapReduceTaskContext.java

```diff
@@ -245,6 +252,14 @@ public void flushOperations() throws Exception {
     }
   }
 
+  public AuthorizationEnforcer getAuthorizationEnforcer() {
+    return authorizationEnforcer;
+  }
+
+  public AuthenticationContext getAuthenticationContext() {
```

Comment:
since these are not overriden methods, would it be good to add a small javadoc (or even code comment) on why these need to be exposed?